subscription.go

```diff
@@ -140,6 +154,49 @@ func (r *marathonClient) RegisterSubscription() error {
 	return nil
 }
 
+func (r *marathonClient) registerSSESubscription() error {
+	// Prevent multiple SSE subscriptions
+	if r.subscribedToSSE {
+		return nil
+	}
```

Comment:
It's still could be affected by a race condition. Would it be better to try to use https://golang.org/pkg/sync/#Once? Or that would be totally users fault to achieve the race condition here and the boolean field is enough? :)